service/worker/service.go

```diff
@@ -114,6 +113,7 @@ type (
 		BatcherConcurrency                    dynamicconfig.IntPropertyFnWithNamespaceFilter
 		EnableParentClosePolicyWorker         dynamicconfig.BoolPropertyFn
 		PerNamespaceWorkerCount               dynamicconfig.IntPropertyFnWithNamespaceFilter
+		PerNamespaceWorkerOptions             dynamicconfig.MapPropertyFnWithNamespaceFilter
```

Comment:
Wouldn't we want these configs to also be per worker, the schedule worker would probably have different constraints then the batch worker?

Reply:
We combine all the per-namespace workflows into one worker for efficiency. If it really becomes impossible to find settings that work for the combined worker we could reconsider, but for now we should try to tune them together.